src/scripts/tests/import-open-api.test.ts

```diff
@@ -2151,51 +2151,158 @@ describe("scripts/import-open-api", () => {
         "
       `);
     });
-  });
-  it("should deal with no 2xx response case", () => {
-    const operation: OperationObject = {
-      summary: "List all fields for the use case schema",
-      operationId: "listFields",
-      responses: {
-        "302": {
-          description: "Just redirect",
+
+    it("should inject some customProps", () => {
+      const operation: OperationObject = {
+        summary: "List all fields for the use case schema",
+        operationId: "listFields",
+        tags: ["schema"],
+        responses: {
+          "200": {
+            description: "An array of schema fields",
+            content: { "application/json": { schema: { $ref: "#/components/schemas/FieldListResponse" } } },
+          },
+          default: {
+            description: "unexpected error",
+            content: {
+              "application/json": {
+                schema: { $ref: "#/components/schemas/APIError" },
+                example: { errors: ["msg1", "msg2"] },
+              },
+            },
+          },
         },
-        default: {
-          description: "unexpected error",
-          content: {
-            "application/json": {
-              schema: { $ref: "#/components/schemas/APIError" },
-              example: { errors: ["msg1", "msg2"] },
+      };
+
+      expect(
+        generateRestfulComponent(operation, "get", "/fields", [], [], [], {
+          resolve: "{res => res}",
```

Comment:
![image](https://user-images.githubusercontent.com/1761469/113142401-1ea04100-922b-11eb-9328-72428b6304fd.png)

The available props (for the types review)